Formula/swiftgen.rb

```diff
@@ -28,59 +35,38 @@ def install
 
     # Install SwiftGen in `libexec` (because of our resource bundle)
     # Then create a symbolic link in `bin`
-    system "bundle", "exec", "rake", "cli:install[#{libexec}]"
+    system "bundle", "exec", "rake", "cli:install[#{libexec},true]"
     bin.install_symlink "#{libexec}/swiftgen"
```

Comment:
Maybe we can use `write_exec_script` to `exec libexec/swiftgen`. That will probably avoid confusing the thin binary.